dashboard/modules/serve/serve_rest_api_impl.py

```diff
@@ -213,6 +215,17 @@ def validate_http_options(self, client, http_options):
                     f"updated: {divergent_http_options}."
                 )
 
+        def log_config_change_default_warning(config):
+            for deployment in [
+                d for app in config.applications for d in app.deployments
+            ]:
+                if "max_concurrent_queries" not in deployment.dict(exclude_unset=True):
```

Comment:
Should this be checking "max_ongoing_requests" instead?

Reply:
Yes, thanks for the catch!